pkg/cloud/rgraph/exec/executor_parallel_test.go

```diff
@@ -156,6 +157,32 @@ func TestParallelExecutorErrorStrategy(t *testing.T) {
 	}
 }
 
+type syncTestActions struct {
```

Comment:
You should document this -- it's not obvious why this test object behaves like this?